Terminal.Gui/Configuration/Scope.cs

```diff
@@ -1,76 +1,65 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Linq;
+﻿#nullable enable
 using System.Reflection;
-using static Terminal.Gui.ConfigurationManager;
 
-#nullable enable
+namespace Terminal.Gui;
 
-namespace Terminal.Gui {
+/// <summary>
+///     Defines a configuration settings scope. Classes that inherit from this abstract class can be used to define scopes
+///     for configuration settings. Each scope is a JSON object that contains a set of configuration settings.
+/// </summary>
+public class Scope<T> : Dictionary<string, ConfigProperty> { //, IScope<Scope<T>> {
```

Comment:
(out of scope)

It is typically strongly recommended NOT to inherit from `Dictionary<TKey,TValue>`

Instead, the recommendation is to implement appropriate interfaces and for the custom type to _own_ a dictionary that is only exposed via the interface implementations.